lib/MusicBrainz/Server/Form/User/EditProfile.pm

```diff
@@ -96,13 +96,13 @@ sub validate_birth_date {
     return $field->add_error("invalid date") unless Date::Calc::check_date($year, $month, $day);
 }
 
-sub _requires_email {
+sub _requires_not_limited {
     my ($self, $field) = @_;
-    return $field->add_error(l('Biography and website fields may not be edited until your email address is confirmed.')) unless $self->ctx->user->has_confirmed_email_address;
+    return $field->add_error(l('Spammers are not welcome, go away.')) unless !$self->ctx->user->is_limited;
```

Comment:
Two small things: (1) I think we don't have to translate this message, since it only shows up if they're abusing the form (the relevant fields are hidden), and (2) `unless !` hurt my head, I think an `if` would suffice. :)